.github/markets/pr_template.md

```diff
@@ -23,6 +23,8 @@ Each json file under the [configs](../../configs) folder correspond to their res
 |`perp_pools`   |`PerpPoolConfig`   |false   |Object that contains the configs for Perp Pools   |
 |`wswth_contract`   |`string`   |false   |wSWTH ERC-20 contract.   |
 |`market_banners`   |`MarketBanner[]`   |true   |market banner configs.   |
+| `native_token_contracts_map` | `object` | false    | Map of token denoms to their respective contract addresses on the native chain. |       |
+| `native_depositor_contracts_map`  | `object`                 | false    | Map of token denoms to their respective native depositor contract addresses       
```

Comment:
Change the description to `Map of axelar connection ids to their respective native depositor contract addresses`